### ISSUE TITLE
Locked out of account after migrating to Molly (from Molly UnifiedPush)

### ISSUE THREAD

**Issue by mon4d** [mollyim/mollyim-android#415]
### Guidelines

- [X] Yes, I have searched for existing issues

### Bug Description

I followed the tutorial for migrating from Signal to Molly, but instead migrated from the now deprecated Molly UnifiedPush version.
After asking me for my 
PIN, SMS and Cloudflare captcha I had to reenter my phone number and pin again (still as part of the same authentification process), just to be presented with the 7 days account lock message.

I found issues almost identical, related to the Signal (not molly) app, but those are now 1-2 years old already. People there worked around it by installing a different version, some claimed the issue came from not having google services installed or using a vpn.

I am posting this as bug because I for sure didn't misstype my PIN (there is a notification if I enter it wrong).

I now hope it will work on 7 days, when supposedly the pin is deleted, along with the data (i have backuped the chats at least)

Anyone experiencing something similar?

### Device Model

Pixel 9a

### Android Version

Android 14 Graphene OS

### App Version

v7.26.1-1-FOSS

**Comment by mon4d** [comment]
This is almost identical to what happened:
https://github.com/signalapp/Signal-Android/issues/12835

Only difference is I don't get any missing playstore notification, which would make sense for the foss version.

**Comment by valldrac** [comment]
Yes, this is a recurring issue in Signal. The root cause seems to be the Signal anti-spam system, that is especially aggressive when FCM isn't used. Unfortunately, we can't fix it on our end.

**Comment by iago-lito** [comment]
Well, this has us locked out of our accounts :\ Is there still a working workaround in #12835 ? What *can* we do in the meantime ? I don't want to upgrade to lineage 22 because I'm fearing this could happen again. And my friend is just out of signal rn with no solution.
